src/DynamoSandbox/DynamoCoreSetup.cs

```diff
@@ -62,34 +60,23 @@ public void RunApplication(Application app)
         {
             try
             {
-                //This line validates if the WebView2 Runtime is installed in the computer, if is not we return and then exit Dynamo
+                // This line validates if the WebView2 Runtime is installed in the computer before launching DynamoSandbox,
+                // if is not we return and then exit Dynamo Sandbox
                 if (!WebView2Utilities.ValidateWebView2RuntimeInstalled())
```

Comment:
Not sure If we are planning to show the message saying the WebView2 is not installed when running Dynamo over Revit but in that case we should move also this code to SplashScreen.xaml.cs